src/main/java/org/broadinstitute/hellbender/tools/walkers/readorientation/CollectDataForReadOrientationFilter.java

```diff
@@ -93,84 +117,46 @@ public void apply(final AlignmentContext alignmentContext, final ReferenceContex
         // manually expand the window and get the 3-mer for now.
         // TODO: implement getBasesInInterval() in referenceContext. Maybe simplify to getKmer(int k)?
         // TODO: this is still relevant (10/2). I shouldn't mess with the internal state of the ref context object
-        referenceContext.setWindow(1, 1);
+        referenceContext.setWindow(REFERENCE_CONTEXT_PADDING_ON_EACH_SIDE, REFERENCE_CONTEXT_PADDING_ON_EACH_SIDE);
         final String refContext = new String(referenceContext.getBases());
-        assert refContext.length() == 3 : "kmer must have length 3";
+        Utils.validate(refContext.length() == REFERENCE_CONTEXT_SIZE, "kmer must have length " + REFERENCE_CONTEXT_SIZE); // or should we return?
```

Comment:
Use `Utils.validate(condition, () -> . . .)` -- that way the error string is only computed if the condition fails.  It's not critical here but it's a good habit to be in.